viewer/core/src/glsl/post-processing/outline-detect-combine.frag

```diff
@@ -71,6 +71,17 @@ void main() {
   float backDepth = texture2D(tBackDepth, vUv).r;  
   float customDepth = texture2D(tCustomDepth, vUv).r;
   float ghostDepth = texture2D(tGhostDepth, vUv).r;
+
+  // This is a hack to make sure that all textures are initialize
+  // If a texture is unused, it will have a clear value of 0.0

```

Comment:
```suggestion
  // This is a hack to make sure that all textures are initialized
  // If a texture is unused, it will have a clear value of 0.0.
  // Without this we've seen issues with MSAA where resizing render targets
  // causes depth to cleared to either 1 or 0 depending on the device/browser
```